cms/core/forms.py

```diff
@@ -98,3 +100,33 @@ def clean_corrections(self) -> StreamValue:  # noqa: C901
 
 class NoLocaleFilterInChoosersForm(SearchFilterMixin, BaseFilterForm):
     """A chooser filter form that deliberately excludes the locale filter."""
+
+
+class PageWithEquationsAdminForm(WagtailAdminPageForm):
```

Comment:
we will need a system check that ensures any Page model that has an equation block has an admin form that inherit from this

Reply:
Addressed in [23af0db](https://github.com/ONSdigital/dis-wagtail/pull/192/commits/23af0db6f4f4a2b6076fb77ba86edb072340cec0)